spec/models/target_spec.rb

```diff
@@ -8,6 +8,22 @@
     it { should validate_presence_of(:latitude) }
     it { should validate_presence_of(:longitude) }
     it { should validate_presence_of(:user) }
+
+    context 'when a user has 10 targets and tries to create another one' do
+      let!(:user_1) { create :user }
```

Comment:
Why do you need a `user_1`. Try to avoid this naming of variables.

Reply:
You can pass association objects to a factory
`let(:target) { create(:target, user: user) } `